sc2reader/engine/plugins/context.py

```diff
@@ -63,7 +63,9 @@ def handleCommandEvent(self, event, replay):
             self.logger.error("Other unit {0} not found".format(event.other_unit_id))
 
     def handleTargetUnitCommandEvent(self, event, replay):
-        self.last_target_ability_event[event.player.pid] = event
+        if event.player is not None:
```

Comment:
```suggestion
        if event.player:
```
All other falsey values (0, 0.0, False, [], {}, tuple(), set()) are just as dangerous as `None`.

Reply:
Thanks @cclauss  - You're right